.dockerignore

```diff
@@ -1,13 +1,8 @@
-**/node_modules
 target
-npm-debug.log
-Dockerfile*
-docker-compose*
-.dockerignore
-.git
-.gitignore
-README.md
-LICENSE
-.vscode
 !target/debug/driver
 !target/release/driver
+dex-contracts
+!dex-contracts/build
+**/node_modules
+.git
+.vscode
```

Comment:
I cleaned this up because the point of the dockerignore file is to make docker have to transfer less data to the docker daemon that builds container. For example if we didn't exclude target it would have to send the whole target folder to the docker daemon which can be quite large.
For small files/folders this doesn't matter unless we were copying the whole root folder into the container which are not doing anywhere.